### PR TITLE
style: implement new getting started page

### PR DESCRIPTION
**Pull request by AlejandroYanes**
fixes: #417

### CODE REVIEW

**Comment by TahimiLeonBravo** [comment]
@AlejandroYanes Now we have this problem in each page of the getting started. The button Show Code does not make sense here. Please review the images
screen shot 2018-12-11 at 2 06 42 pm
screen shot 2018-12-11 at 2 06 53 pm
screen shot 2018-12-11 at 2 07 01 pm
screen shot 2018-12-11 at 2 07 06 pm
screen shot 2018-12-11 at 2 29 00 pm

File: library/pages/components/GettingStartedPage/index.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+                    <RenderIf isTrue={activeTabName === 'usage'}>
+                        <h3 className="section-heading">
+                            There are several runnable examples in this Git repo,
+                            but here's a Hello World one:
+                        </h3>
+                        <CodeEditor code={HelloWorldExample} />
+                        <iframe
+                            title="codesandbox example"
+                            src="https://codesandbox.io/embed/24p8n0pnz0?hidenavigation=1&fontsize=14&view=preview"
+                            style={this.sampleStyles}
```
Comment (LeandroTorresSicilia) [inline library/pages/components/GettingStartedPage/index.js]: why not use className? if somewhere a className is not possible then use style.js file. Remember separation of concerns.

File: library/pages/components/GettingStartedPage/index.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                            </h5>
+                        </div>
+                    </RenderIf>
+                </section>
+            </div>
+        );
+    }
+}
+
+GettingStartedPage.propTypes = {};
```
Comment (LeandroTorresSicilia) [inline library/pages/components/GettingStartedPage/index.js]: remove if there is not props

File: library/pages/components/GettingStartedPage/index.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+GettingStartedPage.defaultProps = {};
```
Comment (LeandroTorresSicilia) [inline library/pages/components/GettingStartedPage/index.js]: same ^^

File: library/pages/components/GettingStartedPage/styles.css [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    padding-right: 24px;
+}
+
+.react-rainbow-getting-started_container {
+    background-color: white;
+    padding: 1.25rem 2rem 0 2rem;
+}
+
+
+.section-heading {
```
Comment (LeandroTorresSicilia) [inline library/pages/components/GettingStartedPage/styles.css]: use css convention, same to all css below

File: library/styleguideComponents/Playground/index.js [hunk 5]
```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './styles.css';
 
+// const pages = [];
```
Comment (LeandroTorresSicilia) [inline library/styleguideComponents/Playground/index.js]: remove if it is not used

**Pull request merged**